mmrazor/models/task_modules/estimators/counters/flops_params_counter.py

```diff
@@ -172,7 +171,8 @@ def params_units_convert(num_params, units='M', precision=3):
         >>> params_units_convert(3e-9)
         '3e-09'
     """
-
+    if getattr(num_params, 'requires_grad', None):
```

Comment:
If you want to get num_params without units, just pass the argument `units` with the `None`.